content/schemas/sign_request_create_signer.yml

```diff
@@ -0,0 +1,114 @@
+---
+title: Signer fields for Create Sign Request
+
+type: object
+
+description: |-
+  The schema for a Signer in a POST Sign Request request body
+
+required:
+  - email
+
+properties:
+  email:
+    type: string
+    description: Email address for signer
+    example: example@gmail.com
+
+  first_name:
+    type: string
+    description: First name of signer
+    example: Aaron
+
+  last_name:
+    type: string
+    description: Last name of signer
+    example: Levie
```

Comment:
This first/last is pretty westernized. How does this deal with 4 names or eastern cultures where surname comes first or Spanish where both parents' names come last but the order varies? Seems simpler to just have one field for name.

Reply:
checking with the sign team on this to see if they could support a single name field!

Reply:
This is being tracked here https://jira.inside-box.net/browse/SIGN-312 it's unclear if we will be able to make that update on the sign request side before GA